File: app/src/main/java/com/etsoft/scales/test1.java
package com.etsoft.scales;

import android.util.Log;

import java.io.BufferedInputStream;
import java.io.IOException;
import java.io.InputStream;
import java.util.Timer;
import java.util.TimerTask;

/**
 * Author：FBL  Time： 2018/10/7.
 */
public class test1 {

    public static void main(String[] args) throws IOException {

        new Timer().schedule(new TimerTask() {
            @Override
            public void run() {
                System.out.println("时间戳 :" + System.currentTimeMillis() + "测试次数");
                cancel();
            }
        }, 0, 2000);

        byte[] buffer = new byte[1024];
        InputStream inStream = new BufferedInputStream(new InputStream() {
            @Override
            public int read() throws IOException {
                return 0;
            }
        });
        while ((inStream.read(buffer) != -1)){


            //下面做正常读取的操作
        }
    }


}

File: app/src/main/java/com/etsoft/scales/test1.kt
package com.etsoft.scales

import android.util.Log

import java.io.BufferedInputStream
import java.io.IOException
import java.io.InputStream
import java.util.Timer
import java.util.TimerTask

/**
 * Author：FBL  Time： 2018/10/7.
 */
object test1 {

    @Throws(IOException::class)
    @JvmStatic
    fun main(args: Array<String>) {

        Timer().schedule(object : TimerTask() {
            override fun run() {
                println("时间戳 :" + System.currentTimeMillis() + "测试次数")
                cancel()
            }
        }, 0, 2000)

        val buffer = ByteArray(1024)
        val inStream = BufferedInputStream(object : InputStream() {
            @Throws(IOException::class)
            override fun read(): Int {
                return 0
            }
        })
        while (inStream.read(buffer) != -1) {


            //下面做正常读取的操作
        }


        Integer.toHexString(255)

        val a: Byte = 62


        java.lang.Byte.toString(a)

    }


}
